src/runtime/vdom/vdom-render.ts

```diff
@@ -192,13 +189,6 @@ const relocateToHostRoot = (parentElm: Element) => {
         // Reset so we can correctly move the node around again.
         childNode['s-sh'] = undefined;
 
-        // When putting an element node back in its original location,
-        // we need to reset the `slot` attribute back to the value it originally had
-        // so we can correctly relocate it again in the future
-        if (childNode.nodeType === NODE_TYPE.ElementNode && !!childNode['s-sn']) {
-          childNode.setAttribute('slot', childNode['s-sn']);
-        }
-
         // Need to tell the render pipeline to check to relocate slot content again
         checkSlotRelocate = true;
       }
```

Comment:
I can't comment on the line itself, but do we still need [this if statement](https://github.com/ionic-team/stencil/pull/5222/files#diff-4828a657f521302051295a582e81299dde7354857095b41f6ae9b062d835670fL233), also introduced in #4993?

Reply:
Yep that can go too. Good call. Merged into commit d2ab3314e7117f370d049dc1b600d69074774328